### PR TITLE
83 store branches

### PR DESCRIPTION
**Pull request by alexcouper**
This solves #83 

I've tested and I do indeed receive new branches in the UI (upon refresh) following a github post :)

### CODE REVIEW

File: ployst/core/client.py [hunk 1]
```diff
@@ -116,11 +116,12 @@ def create_or_update_branch_information(self, branch_info):
             repo=branch_info['repo'],
             name=branch_info['name']
         )
+        print branch_info
```
Comment (txels) [inline ployst/core/client.py:119]: Use a logger debug?

File: ployst/github/github_client.py [hunk 2]
```diff
@@ -16,6 +17,11 @@ def get_secret(org, repo):
             .hexdigest())
 
 
+def compute_github_signature(body, org, repo):
+    secret = get_secret(org, repo)
+    return hmac.new(secret, body, hashlib.sha1).hexdigest()
```
Comment (txels) [inline ployst/github/github_client.py:22]: Nice

File: ployst/github/management/commands/github_poke.py [hunk 3]
```diff
@@ -20,24 +22,29 @@ class Command(BaseCommand):
...
             },
             'ref': branch_name
         })
-        response = requests.post(full_url, data={'payload': load})
+        sig = compute_github_signature(load, org, repo)
+
+        response = requests.post(full_url, data=load,
+                                 headers={'X_HUB_SIGNATURE': sig})
         print response.status_code
+        # TODO: Find out why we're getting a 404 when posting here.
```
Comment (txels) [inline ployst/github/management/commands/github_poke.py:47]: Maybe a `/` too much? _Thanks_ to `ng-resource` we are quite sensitive to those...

Comment (txels) [inline ployst/github/management/commands/github_poke.py:47]: I'm getting `200`.

**Comment by txels** [comment]
This works fine, my PR slightly improves the info we get. I will merge this one and recreate mine to develop.

File: ployst/core/client.py (hunk 1 above)
Comment (alexcouper) [inline ployst/core/client.py:119]: Removing in a new branch

File: ployst/github/management/commands/github_poke.py (hunk 3 above)
Comment (alexcouper) [inline ployst/github/management/commands/github_poke.py:47]: This was a note to myself during development - as you guessed, i resolved it with the removal of the final slash.

**Pull request merged**